website/layouts/index.html

```diff
@@ -55,15 +55,21 @@
         <div class="blog-excerpts">
           <h2>Recently Published</h2>
           <article class="doc">
-            {{ $paginator := .Paginate (where .Site.RegularPages "Type" "in" .Site.Params.mainSections) }}
+            {{ $paginator := .Paginate (where .Site.RegularPages "Type" "in" .Site.Params.mainSections) 2 }}
```

Comment:
What's the `2` for?

Reply:
I'm fuzzy on how it works, but IIUC this part only for the home page article excerpts. There we show the top 2.